### PR TITLE
Design Login

### PR DESCRIPTION
**Pull request by Gimleux**

### CODE REVIEW

File: src/style/helpers.scss [hunk 1]
```diff
@@ -1,3 +1,52 @@
...
+  padding-top: 15px
+}
+
+.login-intro .row {
+  justify-content: center;
+}
+
+.login-input .row:first-child {
+  justify-content: center;
+}
```
Comment (qvalentin) [inline src/style/helpers.scss]: you could also just use the class justify-content-center instead of writing custom css

**Review by qvalentin — CHANGES_REQUESTED** [review]
Looks really great, why no story yet?

File: WebApp.iml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="WEB_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="sourceFolder" forTests="false" />
```
Comment (qvalentin) [inline WebApp.iml]: ???
pls remove or add to gitignore

Comment (Gimleux) [reply WebApp.iml]: Had some problems with the IDE, thought i got every file

File: src/style/helpers.scss (hunk 1 above)
Comment (Gimleux) [reply src/style/helpers.scss]: i think i had some problems using this class (could be because of the containers though). Will have another look at it

**Review by qvalentin — APPROVED** [review]
can be merged

**Pull request merged**